pkg/policy/api/rule_validation.go

```diff
@@ -82,6 +82,62 @@ func (e EgressRule) Validate() error {
 	return nil
 }
 
+// Validate validates Kafka rules
+// TODO we need to add support to check
+// wildcard and prefix/suffix later on.
+func (kr PortRuleKafka) Validate() error {
+	if len(kr.APIKey) > 0 {
+		switch kr.APIKey {
+		case KafkaProduceReq:
+		case KafkaFetchReq:
+		case KafkaCreateTopicsReq:
+		case KafkaDeleteTopicsReq:
+		default:
+			return fmt.Errorf("Invalid Kafka APIKey :", kr.APIKey)
```

Comment:
All errors returned should start with lower capital letters so it can be embedded.

Reply:
If you’r using Errorf, use a proper format string:
`fmt.Errorf(“...: %q”, kr.APIKey)`

Reply:
done